index.html

```diff
@@ -42,7 +44,7 @@ <h3 class="profession">A Web Developer</h3>
         <div class="design"><img src="./images/design.png" alt="design" /></div>
```

Comment:
Kindly, the semantic SECTION tags are used to define the themes of your site, for example, home, projects, about, so using a SECTION inside another SECTION is not a good practice, I recommend changing it to a DIV

Reply:
Kindly, to make your site more accessible I recommend you to follow the order of the titles H1, followed by H2, etc.

Reply:
Ok